ivy/functional/backends/paddle/utility.py

```diff
@@ -37,6 +37,22 @@ def any(
     keepdims: bool = False,
     out: Optional[paddle.Tensor] = None,
 ) -> paddle.Tensor:
-    raise IvyNotImplementedException()
+    x = paddle.to_tensor(x).type(paddle.bool)
```

Comment:
paddle tensors don't have `.type` method, pass the type to the `dtype` argument of `.to_tensor` instead